### PR TITLE
Layout user info

### PR DESCRIPTION
**Pull request by ghost**
## Description
lay out user info.
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

### CODE REVIEW

**Comment by at-thaitran** [comment]
LGTM

File: routes/web.php [hunk 1]
```diff
@@ -74,4 +74,8 @@
         'uses'=>'HomeController@index',
         'as'=>'user'
     ]);
+    Route::get('info', [
```
Comment (at-tuanphan) [inline routes/web.php]: info => profile đi e nghe hay hơn đó

Comment (at-tuanphan) [inline routes/web.php]: dùng lun route::resource lun cho khoẻ em

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**